interop-testing/src/main/proto/io/grpc/testing/integration/empty.proto

```diff
@@ -30,7 +30,7 @@
 
 syntax = "proto2";
 
-package grpc.testing;
+package grpc.testing.integration;
```

Comment:
These protos are copy of protos of in https://github.com/grpc/grpc/tree/master/src/proto/grpc/testing and need to always stay in sync.